.github/workflows/mirror-tooling-images.yml

```diff
@@ -0,0 +1,59 @@
+name: "[Daily] Mirror Tooling Images to GHCR"
+
+on:
+  schedule:
+    - cron: "0 3 * * *" # every day at 3 AM UTC
+  workflow_dispatch:
+    inputs:
+      ghcr_namespace:
+        description: "GHCR Namespace"
+        required: true
+        default: "ghcr.io/project-copacetic/copacetic"
+
+env:
+  GHCR_NAMESPACE: ${{ github.event.inputs.ghcr_namespace || 'ghcr.io/project-copacetic/copacetic' }}
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    strategy:
+      matrix:
+        images:
+          - "debian:10-slim"
+          - "debian:11-slim"
+          - "debian:12-slim"
+          - "debian:stable-slim"
+          - "ubuntu:20.04"
+          - "ubuntu:22.04"
+          - "ubuntu:latest"
```

Comment:
is this one being used?

Reply:
only in the workflows so I've removed this one